Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,16 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            const taskProps: { [key: string]: any; } = {MiniMatchLines: sourceFilter.length};
+            ci.publishTelemetry('TestExecution','MinimatchFilterPerformance', taskProps);
+            var start = new Date().getTime();
             const sources = tl.findMatch(this.inputDataContract.TestSelectionSettings.SearchFolder, sourceFilter);
             tl.debug('tl match count :' + sources.length);
+            var end = new Date().getTime();
```

Comment:
no need to store end, directly compute the difference here

Reply:
Changed it